.github/workflows/release.yaml

```
@@ -306,6 +310,13 @@ jobs:
           name: package-${{ matrix.platform }}
           retention-days: 1
 
+      - name: Upload web build artifact for Github Pages deployment
+        if: ${{ env.is_platform_enabled == 'true' && matrix.platform == 'web' && inputs.deploy-to-pages }}
+        id: deployment
```

```suggestion
        id: pages
```
`deployment` is not descriptive enough in the context of the full `release.yaml`.